### PR TITLE
Add more defaults to ThermalStandard

### PR DESCRIPTION
**Pull request by raphaelsaavedra**
Add more default values to the `ThermalStandard` fields. I added the same values as the null ones.

Please let me know if editing the JSON is all I need to do for this change.

### CODE REVIEW

File: src/descriptors/power_system_structs.json [hunk 1]
```diff
@@ -1983,12 +1988,14 @@
...
-          "validation_action": "error"
+          "validation_action": "error",
+          "default": "nothing"
         },
         {
           "name": "operation_cost",
           "null_value": "ThreePartCost(nothing)",
-          "data_type": "ThreePartCost"
+          "data_type": "ThreePartCost",
+          "default": "ThreePartCost(nothing)"
```
Comment (nickrobinson251) [inline src/descriptors/power_system_structs.json:1998]: does this introduce functionality that relies on the "non-functional" demo constructor
https://github.com/NREL-SIIP/PowerSystems.jl/blob/228ef8dd5765640dee30f7c4e9da924223ea30ab/src/models/generated/ThreePartCost.jl#L47

Would it be preferable to add defaults to `ThreePartCost`?

More generally, i wonder if all `Foo(::Nothing)` methods could be replaced by `Foo()` and appropraite defaults (which would them also be useful to the keyword constructors that override some but not all defaults)

**Comment by jd-lara** [comment]
Not all the fields should have defaults, only those for which there is a reasonable value.

File: src/descriptors/power_system_structs.json [hunk 2]
```diff
@@ -1936,19 +1937,22 @@
...
-          "data_type": "PrimeMovers.PrimeMover"
+          "data_type": "PrimeMovers.PrimeMover",
+          "default": "PrimeMovers.OT"
         },
         {
           "name": "fuel",
           "comment": "prime_mover Fuel according to EIA 923",
           "null_value": "ThermalFuels.OTHER",
-          "data_type": "ThermalFuels.ThermalFuel"
+          "data_type": "ThermalFuels.ThermalFuel",
```
Comment (jd-lara) [inline src/descriptors/power_system_structs.json:1954]: This is a reasonable default.

File: src/descriptors/power_system_structs.json [hunk 3]
```diff
@@ -1936,19 +1937,22 @@
...
-          "needs_conversion": true
+          "needs_conversion": true,
+          "default": "0.0"
         },
         {
           "name": "prime_mover",
           "comment": "prime_mover Technology according to EIA 923",
           "null_value": "PrimeMovers.OT",
-          "data_type": "PrimeMovers.PrimeMover"
+          "data_type": "PrimeMovers.PrimeMover",
```
Comment (jd-lara) [inline src/descriptors/power_system_structs.json:1947]: This is a reasonable default

File: src/descriptors/power_system_structs.json [hunk 4]
```diff
@@ -1903,7 +1903,8 @@
         {
           "null_value": "false",
           "name": "status",
-          "data_type": "Bool"
+          "data_type": "Bool",
```
Comment (jd-lara) [inline src/descriptors/power_system_structs.json:1906]: This should be defined properly by the user

File: src/descriptors/power_system_structs.json [hunk 5]
```diff
@@ -1972,7 +1976,8 @@
             "max": null
           },
           "validation_action": "error",
-          "needs_conversion": true
+          "needs_conversion": true,
```
Comment (jd-lara) [inline src/descriptors/power_system_structs.json:1979]: Same here, these fields should have reasonable values

File: src/descriptors/power_system_structs.json (hunk 1 above)
Comment (jd-lara) [reply src/descriptors/power_system_structs.json:1998]: The same issue is here, there are reasonable estimates for this. Even if it is zero. Having a `nothing` implies that the information doesn't exist or there is no reasonable estimates.

**Comment by jd-lara** [comment]
@raphaelsaavedra you need to run 
```
InfrastructureSystems.generate_structs("./src/descriptors/power_system_structs.json", "./src/models/generated")
``` 

to generate the new ThermalStandard code

File: src/descriptors/power_system_structs.json (hunk 1 above)
Comment (raphaelsaavedra) [reply src/descriptors/power_system_structs.json]: I have removed the default for this field as requested by @jd-lara

File: src/descriptors/power_system_structs.json (hunk 4 above)
Comment (raphaelsaavedra) [reply src/descriptors/power_system_structs.json]: Removed

File: src/descriptors/power_system_structs.json (hunk 5 above)
Comment (raphaelsaavedra) [reply src/descriptors/power_system_structs.json]: Removed

**Pull request merged**